### PR TITLE
[WNMGDS-2212] Many storybook stories updated for easier control via args.

### PR DESCRIPTION
**Pull request by forrestbaer**
## Summary

- Updated a number of stories to use the `text` control format so they can be more easily modified in storybook
- Changed a few `select` types to `radio` to be consistent
- Changed a few `radio` types to `boolean` to be consistent
- Looks like one snapshot update which I don't see any changes in, not sure how that happened but it seems OK.

## How to test

1. `yarn clean && yarn install && yarn build && yarn storybook`
2. evaluate changed stories if you like!

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title` or [NO-TICKET] if this is unticketed work.
- [x] Selected appropriate `Type` (only one) label for this PR, if it is a breaking change, label should only be `Type: Breaking`
- [x] Selected appropriate `Impacts`, multiple can be selected.

### CODE REVIEW

File: packages/design-system/src/components/DateField/MultiInputDateField.stories.jsx [hunk 1]
```diff
@@ -7,11 +7,17 @@ export default {
...
   argTypes: {
-    label: { control: false },
     errorMessage: {
       control: { type: 'text' },
       type: { name: 'string', required: true },
     },
+    label: { control: 'text' },
+    monthDefaultValue: { control: { type: 'number', min: 1, max: 12, steps: 1 } },
+    dayDefaultValue: { control: { type: 'number', min: 1, max: 31, steps: 1 } },
+    yearDefaultValue: { control: 'number' },
```
Comment (pwolfert) [inline packages/design-system/src/components/DateField/MultiInputDateField.stories.jsx:17]: I think in order for setting these to do anything, we need to update the story itself to no longer `useState` but `useArgs`. I've had mixed success with `useArgs`.

File: packages/design-system/src/components/Spinner/Spinner.stories.jsx [hunk 2]
```diff
@@ -9,14 +9,8 @@ export default {
...
-    inversed: {
-      control: 'radio',
-      options: [true, false],
-    },
-    filled: {
-      control: 'radio',
-      options: [true, false],
-    },
+    inversed: { control: 'boolean' },
+    filled: { control: 'boolean' },
```
Comment (pwolfert) [inline packages/design-system/src/components/Spinner/Spinner.stories.jsx:13]: 👍

**Review by pwolfert — APPROVED** [review]
Thanks for cleaning these up. There is a set of controls that aren't functional right now noted below, though.

The visual diff is a really subtle letter-spacing change (Saturday label), and I don't know what caused it. I'm guessing it happened a while ago but the pixel diff was so small that it never changed.

letter-spacing-difference

File: packages/design-system/src/components/DateField/MultiInputDateField.stories.jsx (hunk 1 above)
Comment (forrestbaer) [reply packages/design-system/src/components/DateField/MultiInputDateField.stories.jsx:17]: Oh interesting I didn't realize that didn't update in that case.. I'm just going to remove these then, I don't think they add a lot of value, mostly just wanted to play with the min/max value stepper.

**Pull request merged**